mountpoint-s3/tests/fs.rs

```diff
@@ -700,22 +695,13 @@ async fn test_duplicate_write_fails() {
     assert_eq!(dentry.attr.size, 0);
     let file_ino = dentry.attr.ino;
 
-    let opened = fs
+    let _opened = fs
         .open(file_ino, libc::S_IFREG as i32 | libc::O_WRONLY, 0)
         .await
         .unwrap();
-    _ = fs
```

Comment:
why removing this? "first write should succeed", right?

Reply:
yes, but it's not needed for the test anymore. The open above already resolves this to be a "write" handle, while before we needed an actual write call. What we are testing here, though, is the failure of the next open.

Reply:
ah, I see, the `write` would be redundant for the test with current `open` semantics, thanks